plenum/test/replica/stashing/test_replica_unstashing.py

```diff
@@ -10,30 +10,31 @@ def msg(replica):
                                    view_no=replica.viewNo,
                                    pp_seq_no=replica.last_ordered_3pc[1] + 1,
                                    inst_id=replica.instId)
-    return replica.primaryName, pp
+    return pp, replica.primaryName
 
 
 def test_unstash_catchup(replica, msg):
-    _, pre_prepare = msg
-    replica.stasher.stash(msg, STASH_CATCH_UP)
-    assert replica.stasher.num_stashed_catchup > 0
+    pre_prepare, _ = msg
+    replica.set_view_no(pre_prepare.viewNo + 1)
```

Comment:
Why do we need to set viewno + 1 here?

Reply:
Fixed
https://github.com/hyperledger/indy-plenum/pull/1300/files#diff-14832af9d700da535ee6d89e7691f4e8L18